src/seedu/addressbook/data/AddressBook.java

```diff
@@ -98,6 +99,28 @@ public boolean containsPerson(ReadOnlyPerson key) {
         return allPersons.contains(key);
     }
 
+    
+    
+    /**
+     * Checks if provided theme name exists
+     * */
+    
+    public boolean containsTheme(String theme) {
```

Comment:
This method shouldn't be in the AddressBook class, because themes are not related to AddressBooks at all.